pubspec.yaml

```diff
@@ -9,9 +9,12 @@ environment:
 
 dependencies:
   crypto: ^3.0.3
+  cryptography: ^2.7.0
+  dio: ^5.4.1
   flutter:
     sdk: flutter
```

Comment:
Flutter SDK should not be there since this should be a dart package, not a Flutter one.